### PR TITLE
[FHB-85] Add required staging schema

### PR DESCRIPTION
**Pull request by Top-Cat**

### CODE REVIEW

File: src/service/service-directory-api/src/FamilyHubs.ServiceDirectory.Data/Entities/Staging/ServicesTemp.cs [hunk 1]
```diff
@@ -0,0 +1,8 @@
+﻿namespace FamilyHubs.ServiceDirectory.Data.Entities.Staging;
+
+public class ServicesTemp
+{
+    public long Id { get; set; }
```
Comment (Zac-AND-Digital) [inline src/service/service-directory-api/src/FamilyHubs.ServiceDirectory.Data/Entities/Staging/ServicesTemp.cs:5]: I think the ID here is either a GUID or String (dunno if it results in any performance difference, technically they are Guid's from ORs perspective), as the ID is supposed to be the same as the incoming service's ID e.g., "ac148810-d857-441c-9679-408f346de14b" from the OR example

File: src/service/service-directory-api/src/FamilyHubs.ServiceDirectory.Data/Entities/Staging/ServicesTemp.cs [hunk 2]
```diff
@@ -0,0 +1,8 @@
+﻿namespace FamilyHubs.ServiceDirectory.Data.Entities.Staging;
+
+public class ServicesTemp
+{
+    public long Id { get; set; }
+    public required string Json { get; set; }
```
Comment (Zac-AND-Digital) [inline src/service/service-directory-api/src/FamilyHubs.ServiceDirectory.Data/Entities/Staging/ServicesTemp.cs:6]: .NET 8 supports JSON columns directly, dunno if there's anything special we need to do with the type or if `string` is fine?

Comment (Top-Cat) [reply src/service/service-directory-api/src/FamilyHubs.ServiceDirectory.Data/Entities/Staging/ServicesTemp.cs:6]: My understanding is to do that I'd actually have to provide the full JSON type structure which I'm pretty sure is out of scope of this ticket.

File: src/service/service-directory-api/src/FamilyHubs.ServiceDirectory.Data/Entities/Staging/ServicesTemp.cs (hunk 1 above)
Comment (Top-Cat) [reply src/service/service-directory-api/src/FamilyHubs.ServiceDirectory.Data/Entities/Staging/ServicesTemp.cs]: Fixed

**Review by Zac-AND-Digital — APPROVED** [review]

**Pull request merged**